src/js/components/List/index.d.ts

```diff
@@ -18,7 +18,7 @@ export interface ListProps {
   itemProps?: { [_:string]: { background?: string, border?: BorderType, pad?: PadType}};
   margin?: MarginType;
   onMore?: () => void;
-  onClickItem?: (event: React.MouseEvent) => void;
+  onClickItem?: ((event: {item?: {}, index?: number, datum?: {}, target?: HTMLElement}) => void);
```

Comment:
`target` was just an example, I think we should allow all child props and not just `item`, `index`, or `target`, something to let it include all Mouse Event childs or args.

Reply:
okay I believe that React.MouseEvent covers the children but since we are re-declaring item and index its not included in the React.MouseEvent. Im testing out some of the other children to make sure they are included in the MouseEvent